lib/buffer.js

```diff
@@ -345,6 +365,30 @@ Buffer.prototype.toString = function(encoding, start, end) {
   }
 };
 
+Buffer.prototype.asciiSlice = function(start, end) {
+  return this.toString('ascii', start, end);
+};
+
+Buffer.prototype.base64Slice = function(start, end) {
+  return this.toString('base64', start, end);
+};
+
+Buffer.prototype.binarySlice = function(start, end) {
+  return this.toString('binary', start, end);
+};
+
+Buffer.prototype.hexSlice = function(start, end) {
+  return this.toString('hex', start, end);
+};
+
+Buffer.prototype.ucs2Slice = function(start, end) {
+  return this.toString('ucs2', start, end);
+};
+
+Buffer.prototype.utf8Slice = function(start, end) {
+  return this.toString('utf8', start, end);
```

Comment:
this makes my heart hurt. i used to use these as a more performant way of doing operations. though they are also no loner documented (though IIRC they once were). guess I'll have to switch to using `process.binding('buffer')`.